aten/src/ATen/native/quantized/TensorFactories.cpp

```diff
@@ -58,8 +58,20 @@ Tensor empty_per_channel_affine_quantized(
   TORCH_CHECK(
       options.has_dtype(),
       "Must provide data type for Tensor creation functions.");
+
+  Tensor new_scales;
+  Tensor new_zero_points;
+  if(options.device().type() == DeviceType::CPU){
+    new_scales=scales.cpu();
+    new_zero_points=zero_points.cpu();
+  } else {
+    new_scales=scales.cuda();
```

Comment:
can we get the device from options.device()?
e.g. does
```
new_scales = scales.to(options.device());
```
work?

Reply:
no, at least I couldn't get that to work, all the .to methods require more arguments. The only thing I could get working was the .cpu() and .cuda() functions. I even tried copying the syntax directly from 

https://github.com/pytorch/pytorch/blob/a07b08136fb982a85b24ef09b86a89496247b1a3/aten/src/ATen/templates/TensorBody.h#L643

not sure what the issue is/was.

Reply:
nvm, I'm not sure what I tested before. This has been changed